### PR TITLE
Add section on tooling support for platform standard doc

### PR DESCRIPTION
**Pull request by onovotny**

### CODE REVIEW

**Comment by stephentoub** [comment]
cc: @davidfowl, @Petermarcu

File: Documentation/project-docs/standard-platform.md [hunk 1]
```diff
@@ -236,6 +236,17 @@ The above `project.json` will cause NuGet to do a compatibiltiy check, enforcing
 }
 ```
 
+## Tool Support
+Tooling support for the `netstandard` TFM is as follows. This table will be updated regularly to reflect current status.
```
Comment (mellinoe) [inline Documentation/project-docs/standard-platform.md]: What does "Tooling support" mean? Maybe we can be more fine-grained than that.

Comment (onovotny) [inline Documentation/project-docs/standard-platform.md]: Open to suggestions, this was intended as a starting point. Not sure what the best term would be here.

Comment (davidfowl) [inline Documentation/project-docs/standard-platform.md]: Required Tools

Comment (onovotny) [inline Documentation/project-docs/standard-platform.md]: For both the heading and sentence or just the sentence?

Comment (davidfowl) [inline Documentation/project-docs/standard-platform.md]: Heading

**Comment by onovotny** [comment]
Should I mention Xamarin Studio as not yet/TBD? I assume they'll need to publish an update to consume these packages.

**Comment by Maxwe11** [comment]
Where this list was announced?

**Comment by onovotny** [comment]
@Maxwe11 this is not *yet* an official list. trying to nail that down so people know what they can use

File: Documentation/project-docs/standard-platform.md [hunk 2]
```diff
@@ -236,6 +236,16 @@ The above `project.json` will cause NuGet to do a compatibiltiy check, enforcing
 }
 ```
 
+## Required Tools
+Tooling support for the `netstandard` TFM is as follows. This list will be updated regularly to reflect current status.
+
+- **Visual Studio 2015**: With Update 1 RTM
```
Comment (davidfowl) [inline Documentation/project-docs/standard-platform.md]: Won't be update 1 unfortunately

**Comment by Petermarcu** [comment]
Looks pretty good to me at this point. One comment is that the support really seems tied to the version of NuGet. VS 2012 and 2013 are about Nuget v2 support and when it comes. For VS 2015 its NuGet v3. It could be organized in that way but I'm not sure it matters that much.

**Comment by davidfowl** [comment]
:shipit:

**Comment by onovotny** [comment]
@petermarcu I was trying to be explicit about product support instrof assuming everyone knows v2 vs v3 support.

**Comment by Petermarcu** [comment]
Looks good!

**Pull request merged**